multiaccounts/accounts/database.go

```diff
@@ -804,6 +827,17 @@ func (db *Database) saveOrUpdateAccounts(tx *sql.Tx, accounts []*Account, update
 			}
 			keyUID = &acc.KeyUID
 		}
+		var exists bool
+		err = tx.QueryRow("SELECT EXISTS (SELECT 1 FROM keypairs_accounts WHERE address = ?)", acc.Address).Scan(&exists)
+		// Apply default values if account is new and not a watch only
+		if !exists && acc.Type != AccountTypeWatch {
+			if acc.ProdPreferredChainIDs == "" {
```

Comment:
@alaibe cannot precisely recall, but I swear I saw somewhere that you moved this logic from here to messenger?

Reply:
@saledjenic After check with feature requirement, it shouldn't be possible to have them empty so changing here is working in all cases